components/slider/SliderMark.cs

```diff
@@ -1,5 +1,22 @@
-﻿using Microsoft.AspNetCore.Components;
-using System.Collections.Generic;
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+/* Unmerged change from project 'AntDesign(net5.0)'
+Before:
+using Microsoft.AspNetCore.Components;
+After:
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.AspNetCore.Components;
+*/
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.AspNetCore.Components;
```

Comment:
This

Reply:
Seemed an issue for `dotnet-format`, could be cleaned

Reply:
Thanks, we also want to fix all warnings :)